lib/valkyrie/types.rb

```diff
@@ -53,7 +53,7 @@ module Types
 
     # Represents an array of unique values.
     Set = Array.constructor do |value|
-      ::Array.wrap(value).select(&:present?).uniq.map do |val|
+      ::Array.wrap(value).uniq.map do |val|
```

Comment:
This was meant to get rid of `nil` values - this won't do that anymore, right?

Reply:
@tpendragon there was not test, but yes [nil,nil,true] will be [nil,true]

We could remove nil specifically with nil? (and add a test)

Reply:
:+1: